CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# dbt_netsuite v0.13.0
```

Comment:
could be good to note somewhere that all of these changes only apply to Netsuite2

Reply:
True added!